app/services/PropertyDetailsValuesService.scala

```diff
@@ -127,42 +133,154 @@ trait PropertyDetailsValuesService extends ReliefConstants {
       }
       Future.successful(updatedPropertyDetails)
     }
+
     cacheDraftPropertyDetails(atedRefNo, updatePropertyDetails)
   }
 
+  //TODO This need to be removed in the future
```

Comment:
Again - is there a need to hold onto this code for some reason?

Reply:
No.It has been removed.